[2025-07-04T06:42:58.986+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: merge_query_to_temp_logs.run_merge_query scheduled__2025-07-03T00:00:00+00:00 [queued]>
[2025-07-04T06:42:58.990+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: merge_query_to_temp_logs.run_merge_query scheduled__2025-07-03T00:00:00+00:00 [queued]>
[2025-07-04T06:42:58.990+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-04T06:42:58.997+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_merge_query> on 2025-07-03 00:00:00+00:00
[2025-07-04T06:42:59.001+0000] {standard_task_runner.py:60} INFO - Started process 6280 to run task
[2025-07-04T06:42:59.003+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'merge_query_to_temp_logs', 'run_merge_query', 'scheduled__2025-07-03T00:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/merge_query_dag.py', '--cfg-path', '/tmp/tmpbwf2nj4j']
[2025-07-04T06:42:59.004+0000] {standard_task_runner.py:88} INFO - Job 56: Subtask run_merge_query
[2025-07-04T06:42:59.030+0000] {task_command.py:423} INFO - Running <TaskInstance: merge_query_to_temp_logs.run_merge_query scheduled__2025-07-03T00:00:00+00:00 [running]> on host 0dc0f72acdaa
[2025-07-04T06:42:59.074+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='merge_query_to_temp_logs' AIRFLOW_CTX_TASK_ID='run_merge_query' AIRFLOW_CTX_EXECUTION_DATE='2025-07-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-03T00:00:00+00:00'
[2025-07-04T06:42:59.088+0000] {base.py:83} INFO - Using connection ID 'snowflake_dev_conn' for task execution.
[2025-07-04T06:42:59.089+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-6.2.0-26-generic-x86_64-with-glibc2.34
[2025-07-04T06:42:59.090+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-07-04T06:42:59.548+0000] {cursor.py:1028} INFO - query: [ALTER SESSION SET autocommit=False]
[2025-07-04T06:42:59.735+0000] {cursor.py:1041} INFO - query execution done
[2025-07-04T06:42:59.735+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2025-07-04T06:42:59.736+0000] {sql.py:450} INFO - Running statement: MERGE INTO DEV_DB.DEV_SCHEMA.ATTENDANCE AS target
USING (
    SELECT 1 AS EMP_ID, '2024-07-01'::DATE AS ATTENDANCE_DATE, 'Present' AS STATUS
    UNION ALL
    SELECT 2, '2024-07-01'::DATE, 'Leave'
    UNION ALL
    SELECT 3, '2024-07-01'::DATE, 'Absent'
) AS source
ON target.EMP_ID = source.EMP_ID AND target.ATTENDANCE_DATE = source.ATTENDANCE_DATE
WHEN MATCHED THEN
    UPDATE SET target.STATUS = source.STATUS
WHEN NOT MATCHED THEN
    INSERT (EMP_ID, ATTENDANCE_DATE, STATUS)
    VALUES (source.EMP_ID, source.ATTENDANCE_DATE, source.STATUS);, parameters: None
[2025-07-04T06:42:59.736+0000] {cursor.py:1028} INFO - query: [MERGE INTO DEV_DB.DEV_SCHEMA.ATTENDANCE AS target USING ( SELECT 1 AS EMP_ID, '2...]
[2025-07-04T06:43:00.565+0000] {cursor.py:1041} INFO - query execution done
[2025-07-04T06:43:00.566+0000] {sql.py:459} INFO - Rows affected: 3
[2025-07-04T06:43:00.566+0000] {snowflake.py:400} INFO - Rows affected: 3
[2025-07-04T06:43:00.567+0000] {snowflake.py:401} INFO - Snowflake query id: 01bd7532-3201-c318-000e-50720002f21a
[2025-07-04T06:43:00.567+0000] {cursor.py:1028} INFO - query: [COMMIT]
[2025-07-04T06:43:00.872+0000] {cursor.py:1041} INFO - query execution done
[2025-07-04T06:43:00.874+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2025-07-04T06:43:00.874+0000] {connection.py:718} INFO - closed
[2025-07-04T06:43:00.973+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2025-07-04T06:43:01.112+0000] {merge_query_dag.py:28} INFO - MERGE query executed successfully.
[2025-07-04T06:43:01.113+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-04T06:43:01.133+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=merge_query_to_temp_logs, task_id=run_merge_query, execution_date=20250703T000000, start_date=20250704T064258, end_date=20250704T064301
[2025-07-04T06:43:01.184+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-04T06:43:01.212+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
